src/features/shared/get-contactmomenten-service.ts

```diff
@@ -106,8 +106,12 @@ export function useContactmomentenByZaakUrl(
   page: Ref<number>
 ) {
   function getUrl() {
-    const url = getAllContactmomentenUrl(page.value);
-    url.searchParams.set("objectcontactmomenten.object", self.value);
+    const url = new URL(`${window.gatewayBaseUri}/api/objectcontactmomenten`);
```

Comment:
de getAllContactmomentenUrl wordt nu nog maar op 1 plek gebruikt? misschien omwille van de consistentie en om verder vergissingen te voorkomen, hem als losse functie verwijderen en het binnen useContactmomenenByKlantId op dezelfde manier oplossen als hier gedaan is

Reply:
Goeie, fixed.